src/components/common/Map.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+import { bool } from 'prop-types';
+import { geolocated } from 'react-geolocated';
+
+import marker from '../../img/marker.svg';
+
+const Map = withScriptjs(withGoogleMap(props =>
+  <GoogleMap
+    defaultZoom={18}
+    defaultCenter={{ lat: props.coords.latitude || -34.901112, lng: props.coords.longitude || -56.164532 }}
```

Comment:
Instead of doing every time `props.coords.blabla` you could destructure the object, like so: `const { latitude, longitude } = props.coords;`. This way you will get bot variables and then you can use them in jsx without having to enter the object each time. Here you have more info about it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring